.github/workflows/pr_validation.yml

```diff
@@ -25,13 +25,13 @@ jobs:
             - 'tests/**'
 
     - name: Set up Python 3.8
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4.6.0
       with:
-        python-version: 3.8
+        python-version: 3.8      
     - name: Install dependencies
       run: |
         make init-dev
-        pip install coverage pytest-cov
+        pip install -e .[tests]
```

Comment:
This can be removed, the poetry install inside init-dev does it.
This is from a moment where I was figuring out how to best use poetry with extras/groups